### PR TITLE
This prevents the io.quarkus artifacts being installed.

### PR DESCRIPTION
**Pull request by stuartwdouglas**
Remove io.quarkus artifacts before the cache is uploaded

### CODE REVIEW

File: .github/workflows/ci-actions.yml [hunk 1]
```diff
@@ -99,6 +99,9 @@ jobs:
         with:
           name: maven-repo
           path: maven-repo.tgz
+      - name: Delete Local Artifacts From Cache
+        shell: bash
+        run: rm -r .m2/repository/io/quarkus
```
Comment (gastaldi) [inline .github/workflows/ci-actions.yml:104]: Isn't the path missing a `~`?

Comment (gastaldi) [reply .github/workflows/ci-actions.yml:104]: Ah, the command is already run in `~`, ignore me

**Review by gastaldi — APPROVED** [review]
LGTM

File: .github/workflows/ci-actions.yml [hunk 2]
```diff
@@ -99,6 +99,9 @@ jobs:
         with:
           name: maven-repo
           path: maven-repo.tgz
+      - name: Delete Local Artifacts From Cache
```
Comment (gastaldi) [inline .github/workflows/ci-actions.yml:102]: I wonder if this shouldn't run before the `Cache Maven Repository` action :thinking:

File: .github/workflows/ci-actions.yml (hunk 1 above)
Comment (gastaldi) [reply .github/workflows/ci-actions.yml:104]: Nope, I think I was right, the build failed for this reason :)

File: .github/workflows/ci-actions.yml (hunk 2 above)
Comment (stuartwdouglas) [reply .github/workflows/ci-actions.yml:102]: No, it needs to run after it. 

The cache action is kinda odd, as it adds a step at the end to actually save the result. Basically the initial step restores the cache if it exists, and the 'post' step after the build saves the result.

**Pull request merged**